CHANGELOG.md

```diff
@@ -35,12 +35,21 @@ RELEASING:
 
 ## [Unreleased]
 ### Added
+### Changed
```

Comment:
I've removed by mistake some placeholder comments/explanations which were there under unreleased entries, see https://github.com/GIScience/openrouteservice/pull/1799/files. Do you think we could put them back in? Cheers!

Reply:
We agreed to leave it as is.